PublicMapping/districtbuilder
# Allow admins to archive organization templates

## Overview

Adds the option for admins to archive project templates from the organization home screen. The "..." menu is only available to admins on this page and, when the "delete" option selected, a confirmation modal displays to confirm deletion. Once confirmed, template will be archived and the card is no longer available on organization home screen and templates form.

### Checklist


### Demo
![db_issue1108_demo](https://user-images.githubusercontent.com/36374797/156858595-3a3b06b4-e84c-49cb-9494-32b5393551cc.gif)


### Notes

The confirmation modal used for deleting a template is pretty much the exact same as when deleting a project on the "My Maps" page. It would be worth creating a basic "delete" component used by both (WIP).

## Testing Instructions

- `scripts/server`
- Sign in as an organization admin
- Create a test template from a project
- On the organization home screen confirm "..." menus in each template card
- Select "delete template" and confirm modal appears
- Confirm both "cancel" and "delete" modal buttons function

Closes #1108
